sdk/loadtesting/azure-developer-loadtesting/setup.py

```diff
@@ -1,71 +1,79 @@
-# coding=utf-8
-# --------------------------------------------------------------------------
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is regenerated.
-# --------------------------------------------------------------------------
-# coding: utf-8
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
 
-import os
 import re
-from setuptools import setup, find_packages
-
+import os.path
+from io import open
+from setuptools import find_packages, setup
 
+# Change the PACKAGE_NAME only to change folder and different name
 PACKAGE_NAME = "azure-developer-loadtesting"
-PACKAGE_PPRINT_NAME = "Azure Developer LoadTesting"
+PACKAGE_PPRINT_NAME = "MyService Management"
```

Comment:
this does not look right

Reply:
Sorry - this isn't my change - the bot has added an auto-commit to the PR. I will revert this.

Reply:
I've reverted the auto-commit now. Should be good to go.